api/v1beta1/barbicanapi_types.go

```diff
@@ -64,6 +64,17 @@ type BarbicanAPISpec struct {
 
 	BarbicanAPITemplate `json:",inline"`
 
+	PKCS11 BarbicanPKCS11Template `json:"pkcs11,omitempty"`
+
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:default=Enum=simple_crypto
+	EnabledSecretStores []string `json:"enabledSecretStores"`
+
+	// +kubebuilder:validation:Optional
```

Comment:
This is required - and if we can get an enum of values, that would be awesome.  It is a single string that must be one of "simple_crypto", "pkcs11" ..